hnginternship5/ziki
# Optimized draft code

*#### What does this PR do?*
* it handle backend logic for draft

*#### Description of Task to be completed?*
* Draft logic backend

*#### How should this be manually tested?*
* Open the site on your browser
* Type localhost/drafts
* You should see the draft page

*#### What pages did you work on?*
* Page document.php
* Page routes.php

*#### Screenshots (mandatory unless BE)*
![image](https://user-images.githubusercontent.com/21280968/57011145-1337db80-6bf8-11e9-86eb-d3acf0e29fa6.png)


![image](https://user-images.githubusercontent.com/21280968/57011159-23e85180-6bf8-11e9-9de8-cf336e101af4.png)
